packages/react-instantsearch-dom-maps/src/GeoSearchContext.ts

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+type GeoSearchContextProps = {
+  isRefineOnMapMove: boolean;
+  hasMapMoveSinceLastRefine: boolean;
+  toggleRefineOnMapMove: () => void;
+  setMapMoveSinceLastRefine: () => void;
+  refineWithInstance: () => void;
+}
+
+const GeoSearchContext = React.createContext<Partial<GeoSearchContextProps>>({});
```

Comment:
We should avoid the partial otherwise we'll have to use the `!` operator when we use the context. We can provide default value.